ingestion-core/src/metadata/_version.py

```diff
@@ -7,5 +7,5 @@
 
 from incremental import Version
 
-__version__ = Version("metadata", 0, 9, 0, dev=0)
+__version__ = Version("metadata", 0, 9, 0, dev=1)
```

Comment:
Can you bump this up to `dev=2` as already have `0.9.0.dev1` published - https://test.pypi.org/project/openmetadata-ingestion-core/#history

Reply:
Our logic of bumping up the version may not handle the case when a git revert for a commit that upgrades this is done.

My git revert caused  1 --> 0 roll back - realized this now as I was looking into revert of the revert.